client/components/SearchOperationList.js

```diff
@@ -155,7 +160,7 @@ export default class SearchComponent extends React.Component {
                 store.getOperationTypes().map((t) => <option key={t.id} value={t.id}>{store.operationTypeToLabel(t.id)}</option>)
             );
 
-            details = <div className="panel-body transition-expand">
+            details = <div className="panel-body transition-expand" key={Date.now()}>
```

Comment:
The `this.setState` line in `clearSearch` is supposed to trigger re-rendering, so you shouldn't need this. If it doesn't work once the modifications above have been done, please let me know and I'll give it a look.